index.d.ts

```diff
@@ -21,6 +22,37 @@ declare module 'react-pixi-fiber' {
     children?: React.ReactNode;
   }
 
+  /**
+   * Extra properties to add to allow us to set event handlers using props
+   */
+  export interface InteractiveComponent {
```

Comment:
Is this needed because PIXI defines `InteractionEventTypes` as strings?

Reply:
Yep...

There might be a terser way to accomplish what we need but I wanted to get this working so I could use interactions with TypeScript